File: C#/ToyLanguage/ToyLanguage/Model/Collections/MyHeap.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ToyLanguage.Interfaces;

namespace ToyLanguage.Model.Collections
{
    class MyHeap<K, V> : WrapperDictionary<int, int>, IHeap<int, int>
    {
    }
}

File: Java/src/model/Collections/MyHeap.java
package model.Collections;

import interfaces.IHeap;

import java.util.Map;

/**
 * Created by Lucian on 11/21/2015.
 */
public class MyHeap extends WrapperDictionary<Integer, Integer> implements IHeap<Integer, Integer> {
    //The heap hass same implementation as the dictionary because it is
    //using the key->value structure just with different data types

}
